CMakeLists.txt

```diff
@@ -233,6 +233,9 @@ endif()
   if ( gfortran_compiler AND ( NOT CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 8.0.0 ) )
     add_definitions(-DGCC_GE_8) # Tell library to build against GFortran 8.x bindings w/ descriptor change
   endif()
+  if ( gfortran_compiler AND ( NOT CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 14.0.0 ) )
+    add_definitions(-DGCC_GE_15) # Tell library to build against GFortran 15.x bindings
+  endif()
```

Comment:
At some point I need to refactor the CMake files: we should not be using `add_definitions`, rather we should be specifying properties on targets (that way they can be propagated to dependencies and/or exported)

Reply:
Well, that would be the modern way of using cmake. But also can be a huge effort. A colleague had this project for major ERB software and struggled a long time to renovate the cmake build process. So don't take this step easy.